resources/views/hardware/checkin-due.blade.php

```diff
@@ -0,0 +1,142 @@
+@extends('layouts/default')
+
+@section('title0')
+
+    @if ((Request::get('company_id')) && ($company))
+        {{ $company->name }}
+    @endif
+
+    {{ trans_choice('general.checkin_due_days', $settings->audit_warning_days, ['days' => $settings->audit_warning_days]) }}
+
+@stop
+
+{{-- Page title --}}
+@section('title')
+    @yield('title0')  @parent
+@stop
```

Comment:
This is done elsewhere but I'm not sure why...I think we can put the logic in `title0` directly in `title`?